src/background.ts

```diff
@@ -44,10 +44,12 @@ import initNavigation from "@/background/navigation";
 import initExecutor from "@/background/executor";
 import preload from "@/background/preload";
 import initDeploymentUpdater from "@/background/deployment";
+import initFirefoxCompat from "@/background/firefox-compat";
 
 initNavigation();
 initExecutor();
 initGoogle();
 initFrames();
 preload();
 initDeploymentUpdater();
+initFirefoxCompat();
```

Comment:
Codacy has a fix for the issue: [Promises must be handled appropriately or explicitly marked as ignored with the `void` operator.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7771622)
```suggestion
void initFirefoxCompat();
```